### ISSUE TITLE
Testnet indexing error "greater than maximum value for Int8" for block 22709884 to database failed

### ISSUE THREAD

**Issue by jun0tpyrc** [algorand/indexer#1105]
algorand-indexer:2.12.3 algod:3.8.1, on testnet  
```
err: attemptTx() err: AddBlock() eval err: prepareEvalResources() err: prepareCreators() err: GetAppCreator() err: parseAddress() err: 12181853637140359511 is greater than maximum va
lue for Int8","level":"error","msg":"adding block 22709884 to database failed"
```

upgrade path was from 
2.10.0 => 2.12.3 , supposed no missing important migrations on scheme?

**Comment by rinor** [comment]
indexer-db reports for that round **22709884**
```
2022-07-09 08:44:47.570 UTC [2086923] ERROR:  duplicate key value violates unique constraint "txn_pkey"
2022-07-09 08:44:47.570 UTC [2086923] DETAIL:  Key (round, intra)=(22709884, 0) already exists.
2022-07-09 08:44:47.570 UTC [2086923] CONTEXT:  COPY txn, line 1
2022-07-09 08:44:47.570 UTC [2086923] STATEMENT:  copy "txn" ( "round", "intra", "typeenum", "asset", "txid", "txn", "extra" ) from stdin binary;
```

**Comment by onetechnical** [comment]
This issue is fixed in the 2.12.4 and 2.11.2 releases. Please upgrade to the latest version to resolve this issue.
